tests/testthat/test_time.R

```diff
@@ -0,0 +1,135 @@
+library(testthat)
+library(recipes)
+library(lubridate)
+library(tibble)
+
```

Comment:
Noting that the file name should be `test-time.R`, i.e. created with `use_test()`.